assets/panel/debugger.properties

```diff
@@ -76,11 +76,11 @@ noSourcesText=This page has no sources.
 
 # LOCALIZATION NOTE (noEventListenersText): The text to display in the events tab
 # when there are no events.
-noEventListenersText=No event listeners to display
+noEventListenersText=No event listeners to display.
```

Comment:
For future reference, adding punctuation is not correcting text case. It doesn't help that this was sent into a PR with hundreds of other changes